indexing-service/src/test/java/org/apache/druid/indexing/common/actions/SegmentAllocationQueueTest.java

```diff
@@ -73,6 +89,12 @@ public long getBatchAllocationWaitTime()
       {
         return 0;
       }
+
+      @Override
+      public boolean isSegmentAllocationReduceMetadataIO()
+      {
+        return true;
```

Comment:
Would be nice to run the tests in this class for both true and false and not just true.